x/rollapp/keeper/fraud_proposal.go

```diff
@@ -24,20 +24,27 @@ func (k Keeper) HandleFraud(ctx sdk.Context, rollappID, clientId string, height
 		return err
 	}
 
-	//TODO: mark the rollapp as frozen (if immutable) or mark the fraud height to allow overwriting
-
 	if stateInfo.Sequencer != seqAddr {
 		return sdkerrors.Wrapf(types.ErrInvalidSequencer, "sequencer address %s does not match the one in the state info", seqAddr)
 	}
 
-	// slash the sequencer
+	// slash the sequencer, clean delayed packets
 	err = k.hooks.FraudSubmitted(ctx, rollappID, height, seqAddr)
 	if err != nil {
 		return err
 	}
 
-	//FIXME: make sure the clientId corresponds to the rollappID
+	//mark the rollapp as frozen. revert all pending states to finalized
```

Comment:
we call it "jailed" or "frozen"? terminology wise I think "frozen" is more suitable since "jailed" sounds like a temporary condition and also it adheres to the IBC client / Validator terminology (i.e frozen/jailed respectively)

Reply:
changed to frozen in proto